### DISCUSSION TITLE
modify board._repr_svg_() by passing parameter

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#993] [Q&A] [answered]
**background**
want to modify chess board on google colab using `board._repr_svg_() `

**question**
is there any way to modify chess board by passing parameter on `board._repr_svg_()  `? 

**already try** 
`board._repr_svg_(coordinates = False) ` have an error occured

thanks and best regards

**Comment by niklasf** [comment, marked as answer]
`_repr_svg_()` is a magic method for IPython integration. Instead, use:

```python
import chess.svg

chess.svg.board(board, coordinates=False)
```
